src/contracts/proxies/actions/BasicActions.sol

```diff
@@ -320,7 +320,7 @@ contract BasicActions is CommonActions, IBasicActions {
     _taxSingle(_manager, _safeId);
 
     ISAFEEngine.SAFE memory _safeData = ISAFEEngine(_safeEngine).safes(_safeInfo.collateralType, _safeInfo.safeHandler);
-    _taxSingle(_manager, _safeId);
+
```

Comment:
I understood your message to mean we needed to add this back, not remove it? Is intended to be kept?